### DISCUSSION TITLE
如果想要增加人物，請在此串回報

### DISCUSSION THREAD

**Discussion by jim60105** [sound-buttons/sound-buttons_configs#2] [General]
請提供以下資料，如果有缺漏我會嘗試自己生

- **英文暱稱** (用做網址)
- **正式全名**
- **正面立繪、人設圖** : 至少640*1024，透明png (有多套衣裝的話可全部附上，效果參考[這裡](https://sound-buttons.click/yoruka))
  [參考圖1](https://user-images.githubusercontent.com/16995691/196055399-3cccf435-3dda-4f54-a936-006c263d88dd.png)
  [參考圖2](https://user-images.githubusercontent.com/16995691/196055556-4a6d9aa9-17e2-4be6-b4d3-41297953b147.png)
  [參考圖3](https://user-images.githubusercontent.com/16995691/196055401-25477660-f0d7-46f6-815b-c72ba009ad20.png)
- **主色碼、副色碼** : 主色碼用在按鈕底色，建議為深色 (按鈕是白色字)
- **介紹短文**
- 相關連結 (可選)
    - Youtube
    - twitch
    - twitter
    - facebook
    - discord
    - instagram
    - 其它
- 自介音檔 (可以提供影片出處和秒數、或是直接提供音檔)

感謝所有人的參與

**Comment by Bertram-Chiou** [comment]
英文暱稱 (用做網址) Neko zizi

正式全名 ねこ咨咨 

正面人設圖 : 1024*2048，透明png

![NekoZiZi set](https://user-images.githubusercontent.com/108974488/194115916-51200698-e32d-4d55-a9ff-584425dacb64.png)

主色碼、副色碼

請與 久遠たま 相同

介紹短文

Hello~我是ねこ咨咨
來自南城的幻靈🐱
可以叫我咨咨就好
熱愛玩遊戲、唱歌
自帶天然呆的貓娘~

相關連結 (可選)
Youtube
https://www.youtube.com/channel/UCvwD8Eh3-OapdsoR7Yb8Jhg
Twitch
https://www.twitch.tv/zizi0218520
twitter
https://twitter.com/zi_lydia
DC
https://discord.gg/P6y6DTcyjJ

自介音檔 (可以提供影片出處和秒數、或是直接提供音檔)
https://youtu.be/c24EncYcQpE
06:11  - 06:17

感謝大大

**Reply by jim60105** [reply]
感謝參與，已新增

https://sound-buttons.maki0419.com/zizi

**Reply by Bertram-Chiou** [reply]
感謝大大，想請問這網頁在手機反應會不會較遲緩
ios 使用 safari開啟網頁按鍵點擊後
上方預覽有出現片段
但是網頁不會自動撥放音效

**Reply by jim60105** [reply]
我沒有ios能測試，但是在Android手機是正常的

**Reply by Bertram-Chiou** [reply]
大大你好

咨咨有2.0形象了

可以幫忙新增嗎?

感謝大大
![2noback](https://user-images.githubusercontent.com/108974488/236899265-30b8ad80-d68c-4536-af70-9a5619628b7c.png)

**Reply by jim60105** [reply]
已新增！

**Reply by Bertram-Chiou** [reply]
非常感謝大大

**Comment by kelvin0511** [comment]
### 英文暱稱
Enomu165

### 正式全名
伊糯姆

### 正面立繪、人設圖 : 至少640*1024，透明png
[圖1](https://cdn.discordapp.com/attachments/895667948881064018/1041763665218650123/eno2.png)
[圖2](https://cdn.discordapp.com/attachments/895667948881064018/1041763665797459968/eno3.png)

### 主色碼、副色碼
主: #ffdd82 
副: #ffbbbd

### 介紹短文
來自鼠鼠星球的黃金鼠!目前在鼠鼠學校讀書，畢業條件是到人類世界，並且想辦法獲得人類的認同!所以~請關注我吧!(⁎˃ᆺ˂)
中文⭕日文🔺英文❌伊糯姆語⭕

✂️= = = = = = = = = = = = = = = = = = = = = = = = = = = =

主要配信：遊戲、雜談、唱歌

✂️= = = = = = = = = = = = = = = = = = = = = = = = = = = =

團隊：第二鈕扣風味社
https://twitter.com/second2button

### 相關連結 (可選)
> [Youtube](https://www.youtube.com/channel/UC72QDntJADIJpkbwX0iE7GQ)
`https://www.youtube.com/channel/UC72QDntJADIJpkbwX0iE7GQ`

> [twitter](https://twitter.com/Enomu165)
`https://twitter.com/Enomu165`

> facebook (N.A.)

> [discord](https://discord.gg/HaRAKSSJ8r)
`https://discord.gg/HaRAKSSJ8r`

> instagram (N.A.)

> [其它: POTOFU](https://potofu.me/enomu165)
`https://potofu.me/enomu165`

### 自介音檔 (可以提供影片出處和秒數、或是直接提供音檔)
https://youtu.be/Lifr6n9hE3Y
13:20-13:33

謝謝

**Reply by jim60105** [reply]
感謝參與，已新增
https://sound-buttons.maki0419.com/enomu165

按鈕顏色有點太淺了，我建議換一個深一點的色碼

**Reply by kelvin0511** [reply]
謝謝~ 我直接在JSON檔裡面改可以嗎?

**Reply by kelvin0511** [reply]
https://cdn.discordapp.com/attachments/895667948881064018/1042089955209510932/image.png

然後這裡有看到是有那個字在的可是好像沒顯示出來 謝謝

**Reply by jim60105** [reply]
都OK

**Comment by TiaMiu** [comment]
英文暱稱 (用做網址) Tia Miu
正式全名 Tia Miu

正面立繪、人設圖 : 至少640*1024，透明png (有多套衣裝的話可全部附上，效果參考[這裡](https://sound-buttons.maki0419.com/tama))
[圖1](https://drive.google.com/file/d/14HQyrkJ-f48MvKojiFh3WOi_G4s0GEHl/view?usp=share_link)
[圖2](https://drive.google.com/file/d/1UYtlm3vV279voIpA0iN1phpinwWHjp5q/view?usp=share_link)
[圖3](https://drive.google.com/file/d/1rXrgr3NpnHts3xappV4xCN42EXblkX_S/view?usp=share_link)
[圖4](https://drive.google.com/file/d/1fZkSx8w1OCXE1wSgfDLKfC_AqQe9v6aK/view?usp=share_link)
[圖5](https://drive.google.com/file/d/1VICILAznHQqHBnZXrdDTUfsLUuO__ajC/view?usp=share_link)

主色碼FFBFC9
副色碼744C60

介紹短文
美索不達米亞神話中，提亞馬特轉生之海洋女神，化身成金魚的身分活動中

相關連結
[Youtube](https://www.youtube.com/channel/UC2uiZzfcPQy_wWRzPxYVB_Q/featured)
[twitter](https://twitter.com/TiaMiuuu)

[twitch](https://www.twitch.tv/tiamiuuuu)
[facebook](https://www.facebook.com/people/Miu-Tia/100069967191971/)
[discord](https://discord.gg/vX7zE3eFuj)

**Reply by jim60105** [reply]
感謝參與，已新增
https://sound-buttons.maki0419.com/tiamiu

後兩張Q版的圖比例會跑版，所以只用了前三張圖喔

**Comment by leadra** [comment]
英文暱稱 Shitora
正式全名 涼風しとら
正面立繪、人設圖 : 至少640*1024，透明png (有多套衣裝的話可全部附上，效果參考這裡)
https://shinengumi.com/member/shitora/
[人設1](https://cdn.discordapp.com/attachments/943099687760715796/1059442957410897920/02.png)
[人設2](https://cdn.discordapp.com/attachments/943099687760715796/1059439071216275476/-1185x1536.png)
[新衣裝](https://cdn.discordapp.com/attachments/943099687760715796/1059440479860703262/FdEXVbhagAIgh3Q.png)

主色碼、副色碼 : 6cad35綠 fffd73黃
主色碼用在按鈕底色，建議為深色 (按鈕是白色字)

介紹短文
深淵の神ハスター様のもう一人の眷属。人の世界のアニメやオタク文化に触れ、修行のことを忘れオタクに…　Vtuberで信者を増やすことに目を付けた。
我是深淵組Vtuber 涼風しとら！
人類世界的御宅文化很有趣！

相關連結 (可選)
https://lit.link/en/suzukazeshitora
Youtube
https://www.youtube.com/@SuzukazeShitora
twitter
https://twitter.com/suzukazeshitora
fanbox
https://suzukazeshitora.fanbox.cc/plans
skeb
https://skeb.jp/@suzukazeshitora
booth
https://shinengumi.booth.pm/
facebook深淵組
https://www.facebook.com/shinengumivtg
discord深淵組公式
https://discord.com/invite/75HeU7A5qR


其它
自介音檔 (可以提供影片出處和秒數、或是直接提供音檔)
初配信12:18-中文介紹版
https://youtu.be/SeZK8wxvlkE?t=738 
日文介紹版04:59
https://youtu.be/SeZK8wxvlkE?t=299

----
就麻煩琳了

**Reply by jim60105** [reply]
感謝參與，已新增
https://sound-buttons.maki0419.com/shitora

**Reply by leadra** [reply]
不好意思，想問，第一組的名稱可以改炒飯妹妹嗎
然後副色碼好亮..還是弄內層的暗系列好了976c32
因為不太熟git，我之後是要點shitora.json進去提交修改建議嗎

**Reply by jim60105** [reply]
已修正

可以發PR，不熟git的話可以也直接跟我說，我會幫你改

**Comment by Bertram-Chiou** [comment]
英文暱稱 (用做網址) 
Sapphire

正式全名 
賽菲爾

正面立繪、人設圖 : 

![Sapphire_](https://user-images.githubusercontent.com/108974488/236653103-728e45eb-27b7-435e-a23a-50bce096c065.png)

主色碼、副色碼 : 主色碼用在按鈕底色，建議為深色 (按鈕是白色字) 
請與四条ユリ相同即可

介紹短文
大家好，我是賽菲爾
叫我賽菲或菲爾也可以喔
是正統的寶箱怪女王

相關連結 (可選)
Youtube
https://www.youtube.com/@sapphire4169E1

twitter
https://twitter.com/Sapphire4169e1

facebook
https://www.facebook.com/profile.php?id=100089018445243

Twitch
https://www.twitch.tv/sapphire4169e1

其它
https://sapphire4169e1.mystrikingly.com/

自介音檔 (可以提供影片出處和秒數、或是直接提供音檔)
請問可以建立後補嗎

感謝大大

**Reply by jim60105** [reply]
感謝參與，已新增
https://sound-buttons.maki0419.com/sapphire

**Comment by MengCynia** [comment]
英文暱稱 Cynia
正式全名 欣希亞
正面立繪、人設圖 : 至少640*1024，透明png (有多套衣裝的話可全部附上，效果參考這裡)
[圖1](https://drive.google.com/file/d/1G2zzXojGaRHILnop5EOoyke6AQAfd7b9/view?usp=share_link)
[圖2](https://drive.google.com/file/d/1Mm9UbevYxGDEiPFRPTKTbC8NPts0tNVl/view?usp=share_link)
[圖3](https://drive.google.com/file/d/1PpMqOdNJTUTYbYURKeau1URV94ndip-L/view?usp=share_link)


主色碼、副色碼 : 主色碼用在按鈕底色，建議為深色 (按鈕是白色字)
主色碼#9362aa
副色碼#bf8ad7

介紹短文

> 安安~ 哈囉哈囉~
> 我是台灣Vtuber欣希亞！！
> #16歲上大學的天才少女
> 以成為全職Vtuber為目標努力著！！！
> 生日是4月2日好奇心滿滿的牡羊座(๑•̀ㅂ•́)و✧
> 最喜歡的食物是冰淇淋和巧克力
> 
> 因為小時候很少玩遊戲
> 所以對各種遊戲都很感興趣
> 也會舉辦一些企劃
> 像節日限定活動、台詞回、耐久24小時直播
> 大家有興趣歡迎來找我玩唷~

相關連結

Youtube
https://www.youtube.com/@CyniaLive

Twitch
https://www.twitch.tv/mengcynia

twitter
https://twitter.com/Meng_Cynia

discord
https://discord.com/invite/bhpVt5k

facebook
https://www.facebook.com/Cynia.Live

instagram
https://www.instagram.com/meng_cynia/

potofu
https://potofu.me/meng-cynia



### 音檔 **(粗體字為分組名稱)**

> ### **欣希亞**
>[大家安安~ 哈囉哈囉~](https://drive.google.com/file/d/1Hv5RB1UcIowiQOLDBk03qnw0Rk0zRFMh/view?usp=share_link)
> ### **口頭禪**
>[A秀](https://drive.google.com/file/d/1pljvOhtTo7dIBMno68x20A-qnfZqLyZD/view?usp=share_link)
> ### **唱歌**
> 
> ### **台詞回**
> 
> ### **奇怪的聲音**
> [阿巴阿巴阿巴](https://youtube.com/clip/Ugkx06JPNPIGSyXkqzJgO8Wtv4LOJ0sEE-fW)
> 
> ### **福利**
> 
> ### **蘿希亞**
> 
> ### **姐希亞**
> [ARA~ ARA~ 真是個任性的孩子呢~](https://drive.google.com/file/d/1sGOqbJuekQEWCXsF7EgFBG_KdIubMUeN/view?usp=share_link)
> ### **艾蕾亞**
> 
> ### **白菇姬**
> 
> 

其他分類之後會與觀眾一起補齊
感謝大大🙏💜

**Reply by jim60105** [reply]
已新增 https://sound-buttons.maki0419.com/cynia
第三張Q版的圖，考慮到立繪的統一性所以沒有放

**Comment by Sotehc** [comment]
英文暱稱 Yoshika
正式全名 よしか YOSHIKA⁂
正面立繪、人設圖 : 至少640*1024，透明png (有多套衣裝的話可全部附上，效果參考這裡)
[圖1](https://drive.google.com/file/d/1Ftv835t-jQL9ol2US3DugwLQslfshFO-/view?usp=sharing)
[圖2](https://drive.google.com/file/d/1RaKIvB9FRugf0J9GfVg_NTLRd6jsOcfb/view?usp=sharing)


主色碼、副色碼 : 主色碼用在按鈕底色，建議為深色 (按鈕是白色字)
主色碼#8c6956
副色碼#701514

介紹短文

歌えるさつまいもアイドルを目指し修行中のVtuberです！
日本一のVsingerアイドル？を目標に頑張ります！

相關連結

Youtube
https://www.youtube.com/@YOSHIKA-Ch


twitter
https://twitter.com/yosi_k14



音檔部分後續再編輯補齊。
麻煩協助新增，感謝大大。

**Reply by jim60105** [reply]
感謝參與，已新增
https://sound-buttons.maki0419.com/yoshika

**Comment by satorihana** [comment]
英文暱稱 satorihana
正式全名 佐鳥華
[正面立繪:]
[https://drive.google.com/file/d/12A-7M4K-wpvDwcYZCjc-AR7bXlaJQ_SZ/view?usp=sharing](圖)

主色碼、副色碼 : 主色碼用在按鈕底色，建議為深色 (按鈕是白色字)

與四條ユリ相一致即可
介紹短文
我是修行了150年的蜜袋鼯精道長~
相關連結 (可選)
Youtube: https://www.youtube.com/@satorich.8864

twitter: https://twitter.com/satorihanavt

其它
自介音檔 (可以提供影片出處和秒數、或是直接提供音檔)
[https://www.youtube.com/live/SWZTFLyPsi4?feature=share&t=1267](url)

**Reply by jim60105** [reply]
感謝參與，已新增
https://sound-buttons.maki0419.com/satorihana

**Comment by zeldaluna** [comment]
英文暱稱 (用做網址)
Airuier
正式全名
東域艾瑞兒
正面立繪、人設圖 : 至少640*1024，透明png (有多套衣裝的話可全部附上，效果參考這裡)
https://i.imgur.com/nqRNvte.png

https://i.imgur.com/s85mWvI.png
主色碼、副色碼 : 主色碼用在按鈕底色，建議為深色 (按鈕是白色字)
#DF90BA
介紹短文
東域艾瑞兒,是Vtuber。不是鵝也不是你媽,虛擬美少女。
性感美麗知性主播,請悉知。
相關連結 (可選)
Youtube
https://youtube.com/@Airuier
twitch
https://www.twitch.tv/eastairuier
twitter
https://twitter.com/EastAiruier
facebook
https://www.facebook.com/profile.php?id=100068916579216
discord
https://discord.gg/9Beemnfj
instagram
https://instagram.com/airuier_vtuber

自介音檔 (可以提供影片出處和秒數、或是直接提供音檔)
https://www.youtube.com/live/5bQ9IBK9r84?si=VQlP78pqsDpdBzqz&t=651

感謝平台 感謝作者

**Reply by jim60105** [reply]
感謝參與，已新增
https://sound-buttons.maki0419.com/airuier

**Reply by zeldaluna** [reply]
感謝🙏🙏🙏

**Comment by syachiku0220** [comment]
英文暱稱 (用做網址)
HanazonoSayaka

正式全名
花園さやか

正面立繪、人設圖 :
<img width="800" height="1200" alt="Hanazono-Sayaka_pr-img_01" src="https://github.com/user-attachments/assets/c79278f7-3da6-48e1-999b-9183a6e94b6b" />

主色碼、副色碼 : 主色碼用在按鈕底色，建議為深色 (按鈕是白色字)
#D3B674/ #ECC45F

介紹短文
たくさんの情報を分かりやすく整理整頓し、自分の言葉で伝えられるようになることが目標。
頭を使う(?)ためか甘いものが大好きで、おやつは欠かさず持ち歩いている。
SNSに投稿される視聴者さんのホロ活報告を、こっそり眺めるのが最近の幸せ。

相關連結 (可選)
Youtube https://www.youtube.com/watch?v=voYUTS9uLh0&list=PLYWHLZk82krsdAKXxA-ap7ptsy-0Q6QqG

感謝大大

**Reply by jim60105** [reply]
感謝參與，已新增
https://sound-buttons.click/hanazonosayaka

最近我這裡正好在進行基礎設施遷移，可能會不太穩定
若是遇到奇怪的問題請跟我說

**Reply by syachiku0220** [reply]
好的感謝
